app/hotels/src/allHotels/searchForm/locationPicker/LocationPickerScreen.js

```diff
@@ -0,0 +1,185 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import {
+  Text,
+  Touchable,
+  StyleSheet,
+  Color,
+  TextInput,
+} from '@kiwicom/react-native-app-shared';
+import { type NavigationType } from '@kiwicom/react-native-app-navigation';
+import { PublicApiRenderer } from '@kiwicom/react-native-app-relay';
+import { graphql } from 'react-relay';
+
+import type { LocationPickerScreen_cities_QueryResponse as LocationSuggestions } from './__generated__/LocationPickerScreen_cities_Query.graphql';
+import SuggestionList from './SuggestionList';
+
+type Props = {|
+  navigation: NavigationType,
+|};
+
+type NavigationProps = {|
+  cityId: string,
+  cityName: string,
+  navigation: NavigationType,
+|};
+
+type State = {|
+  search: string,
+  cityId: string,
+|};
+
+const styles = StyleSheet.create({
+  cancelButton: {
+    padding: 8,
+  },
+  headerButtonText: {
+    fontSize: 17,
+    color: Color.white,
+    lineHeight: 20,
+    fontFamily: 'SFProText-Regular',
+  },
+  confirmButtonText: {
+    fontFamily: 'SFProText-Semibold',
+  },
+  headerTitle: {
+    color: Color.white,
+    fontFamily: 'SFProText-Semibold',
```

Comment:
What about Android?

Reply:
There really is no design for android. The font will work on android, since it is custom font. Should I rather use default font on android?

Reply:
It's definitely not SF* font on Android.

Reply:
Ok, I will change to default font then

Reply:
Well, at least setup this only for iOS. We use Roboto on Android.